HLX503/iocBoot/iocHLX503-IOC-01/config.xml

```diff
@@ -19,17 +19,18 @@
     <macro name="1KPOTHE3POTLO_CHANNEL" pattern="^[1-3]$" description="The channel to use when controlling the 1K Pot/He3Pot at low temperatures." hasDefault="YES" defaultValue="2" />
 
     <macro name="DEFAULT_POST_CONDENSE_TEMP_SETPOINT" pattern="^.*$" description="The default temperature setpoint to set just after a recondense" hasDefault="YES" defaultValue="0.3"/>
-    <macro name="RECONDENSE_SORB_TEMP_SET" pattern="^.*$" description="The temperature setpoint to set for the sorb in a recondense" hasDefault="YES" defaultValue="33.0"/>
+    <macro name="RECONDENSE_SORB_TEMP_SET" pattern="^.*$" description="The temperature setpoint to set for the sorb in a recondense" hasDefault="YES" defaultValue="35.0"/>
     <macro name="RECONDENSE_SORB_P" pattern="^.*$" description="The proportional to set for the sorb in a recodense" hasDefault="YES" defaultValue="10"/>
     <macro name="RECONDENSE_SORB_I" pattern="^.*$" description="The integral to set for the sorb in a recodense" hasDefault="YES" defaultValue="1"/>
     <macro name="RECONDENSE_SORB_D" pattern="^.*$" description="The derivative to set for the sorb in a recodense" hasDefault="YES" defaultValue="0"/>
     <macro name="RECONDENSE_SORB_TEMP_FINAL" pattern="^.*$" description="The final temperature of the sorb to wait for in part 3" hasDefault="YES" defaultValue="20.0"/>
-    <macro name="RECONDENSE_HE3POT_TARGET_TEMP_PART1" pattern="^.*$" description="The target temperature of the He3Pot to wait for in part 1" hasDefault="YES" defaultValue="1.8"/>
+    <macro name="RECONDENSE_HE3POT_TARGET_TEMP_PART1" pattern="^.*$" description="The target temperature of the He3Pot to wait for in part 1" hasDefault="YES" defaultValue="1.95"/>
     <macro name="RECONDENSE_HE3POT_TARGET_TEMP_PART2" pattern="^.*$" description="The target temperature of the He3Pot to wait for in part 2" hasDefault="YES" defaultValue="1.6"/>
     <macro name="RECONDENSE_POST_PART2_WAIT_TIME" pattern="^.*$" description="The time in seconds to wait after part 2 of a recondense" hasDefault="YES" defaultValue="10"/> 
     <macro name="RECONDENSE_PART_TIMEOUT"  pattern="^.*$" description="The time to wait before timing out a part in a recondense." hasDefault="YES" defaultValue="3600"/> 
     <macro name="RECONDENSE_PART_TIMEOUT_ON"  pattern="^.*$" description="Whether to timeout parts of the recondense." hasDefault="YES" defaultValue="YES"/> 
-    <macro name="MAX_TEMP_FOR_HE3_COOLING" pattern="^.*$" description="The temperature below which He3 cooling is to be used." hasDefault="YES" defaultValue="1.7"/>
+    <macro name="MAX_TEMP_FOR_HE3_COOLING" pattern="^.*$" description="The temperature below which He3 cooling is to be used." hasDefault="YES" defaultValue="1.75"/>
+    <macro name="SENSOR_SWITCHING_THRESHOLD" pattern="^.*$" description="The temperature threshold to switch between hi and lo sensors." hasDefault="YES" defaultValue="1.45"/>
```

Comment:
I'm now getting some IOC test framework failures as this macro isn't set in the tests. Couldn't see a PR for the tests - did you forget to push a branch?

Reply:
hmm, all of mine pass using master still. I haven't touched the tests, but the macros should have default values?

Reply:
ok cheers, have just pushed that